dubbo-common/src/main/java/org/apache/dubbo/common/URL.java

```diff
@@ -1448,11 +1448,7 @@ public boolean equals(Object obj) {
             return false;
         }
         URL other = (URL) obj;
-        if (host == null) {
-            if (other.host != null) {
-                return false;
-            }
-        } else if (!host.equals(other.host)) {
+        if(!StringUtils.isEquals(host, other.host)) {
             return false;
         }
         if (parameters == null) {

```

Comment:
what about this block, should we refactor at the same time?

Reply:
Should I split them into 2 different commits? Do you mean that?

Reply:
oh, seems `parameters` is a Map, never mind.